packages/viz/src/Source.ts

```diff
@@ -0,0 +1,25 @@
+class Source {
```

Comment:
Yeah, I also think this Source will be useful as a separated class.

One thing that comes to my mind is that it might even have 'data-related' methods, by using internally an SQL client reference ('sql' api package). Thinking about 'getGeometryType', 'getBounds', 'getStats'....

Reply:
That might make sense! We can do that